app/models/manageiq/providers/network_manager.rb

```diff
@@ -87,14 +45,5 @@ def self.supported_types_and_descriptions_hash
         hash[klass.ems_type] = klass.description
       end
     end
-
-    def name
-      "#{parent_manager.try(:name)} #{PROVIDER_NAME}"
-    end
-
-    def self.find_object_for_belongs_to_filter(name)
-      name.gsub!(" #{self::PROVIDER_NAME}", "")
-      includes(:parent_manager).find_by(:parent_managers_ext_management_systems => {:name => name})
```

Comment:
I'm going to need some help with this one, I'm not sure why this is needed for the NetworkManager but not other typically child managers like the storage manager.

It shouldn't have to belong in the base NetworkManager class but this doesn't appear to work if it is in the provider subclasses.